modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeDefinitionLocalServiceImpl.java

```diff
@@ -71,6 +73,7 @@ public ListTypeDefinition addListTypeDefinition(
 			List<ListTypeEntry> listTypeEntries)
 		throws PortalException {
 
+		_validateInvokerBundle(system);
```

Comment:
It seems that we also need to validate the InvokerBundle in the other `addListTypeDefinition(String externalReferenceCode, long userId)` method above. What do you think?

Reply:
In this method, we don't need to validate because system is always false.